### ISSUE TITLE
Lomiri+ --- Dark mode, Side-Stage, App Drawer Problems

### ISSUE THREAD

**Issue by bigbrotherisstillwatching** [kugiigi/jerk-packages#22]
Hello Kugiigi,

there are a few problems i have with the Lomiri+-package.

I use Ubuntu Touch Focal OTA-2 on a Fairphone 4.

1. Sometimes after rebooting the phone, Dark Mode is turned on, even if i had it disabled before restarting. This happens only sometimes. I don't know exactly, but the time of day could be the cause. I have disabled Auto Dark Mode.

2. When disabling Side-Stage and swiping from the right edge to the middle to see the running applications, the background of the running apps selector is black and not transparent as usual. This is annoying and makes no sense to me.

3. The App Drawer will not renew even after installing or uninstalling Waydroid- or click-packages. It sometimes does, but without any apparent reason. After rebooting the phone it is renewed correctly. Btw. i have activated "Reset view upon opening". The factory solution to renew the app list manually by swiping is way better, more reliable and makes more sense to me.

Kind Regards!

bbisw

**Comment by kugiigi** [comment]
Hi,
1. Please observe more and make sure you actually have Auto Dark Mode disabled when this happens. I have no idea how this should happen.
2. I was able to replicate this. Thanks for reporting. 
3. This is probably a bug in Lomiri. The "Reset view upon opening" option is only to scroll to the top/bottom depending if it's inverted or not. You can still refresh the app list by dragging from the top.

**Comment by kugiigi** [comment]
Item#2 is possibly fixed by #23

**Comment by bigbrotherisstillwatching** [comment]
OK, i will test the fix and further observe the other two problems. Give me a few days. I'll answer then. And i think i've found another error:

- when uninstalling Lomiri+ (with `jerk uninstall ***.tar.gz`) ota updates stay disabled, like it should be
- when doing `jerk unblock-ota` after that, i get `Segmentation fault`
- will try to replicate this once more, because i only tried this once

**Comment by bigbrotherisstillwatching** [comment]
OK, before going on with testing, i have to report to you, that the file: Greeter/GreeterView.qml is not present in the original /usr/share/lomiri folder.

I will continue testing, when this is clarified!

**Comment by kugiigi** [comment]
> OK, before going on with testing, i have to report to you, that the file: Greeter/GreeterView.qml is not present in the original /usr/share/lomiri folder.
> 
> I will continue testing, when this is clarified!

Sorry but the latest version (one with the fix) is only compatible with focal devel and probably RC too. If you're on stable, you may have to wait for OTA3.

**Comment by bigbrotherisstillwatching** [comment]
Good to know. Yes i'm on stable, so i'll have to wait.

Then let's go on with the other things i've mentioned:

1. I can confirm, that when rebooting between 8 PM and 7 AM Dark Mode gets turned on automatically. I'm sure, that it was turned off before rebooting and Auto Dark Mode is definitely turned off too. Btw my timezone is Berlin.

2. Like you mentioned only on Focal-devel.

3. After removing Lomiri+ completely i can refresh the app list without problems by swiping down from the top. It always works. But with Lomiri+ installed i cannot refresh by swiping down. It sometimes (!) refreshes by itself without swiping manually, but without any apparent reason, like i mentioned above. Maybe this is a bug, i don't know.

4. I can confirm, what i've written in Post #4. The error message is:

`./jerk unblock-ota
OTA updates will now work again. Run 'jerk block-ota' to block OTA updates.
Instalation requires to run a command to take effect (Command: lomiri-app-stop lomiri-system-settings && lomiri-app-launch lomiri-system-settings). Run now? (y/Y): y
./jerk: line 404: 698527 Segmentation fault      lomiri-app-launch lomiri-system-settings`

The "OTA is blocked by jerk" message in system settings is gone and it shows me updates for click packages after that, but i think Segmentation fault is not good though.

Greetings bbisw

**Comment by bigbrotherisstillwatching** [comment]
Hello again,

i recently switched to OTA-5 and tried the new Lomiri Plus. There are problems with it:

1. The Dark Mode problem mentioned above was still there in OTA-4. I don't know if it's fixed in the new Lomiri Plus version. The only workaround i've found was to enable Auto Dark Mode and set start AND stop time to e.g.: 06:00.

2. Haven't tested with OTA-5 and the new Lomiri+ version, but i think it was gone with OTA-4 and the next-to-last Lomiri+ version.

3. Although it was gone with OTA-4 and the next-to-last Lomiri+ version it again doesn't work with OTA-5.

4. I don't use the jerk-installer anymore, so i don't know.

5. There's a new problem with the Lomiri+ settings. Components->Launcher->Show Launcher when no app is running does not work. The launcher stays hided even when no app is running.

Btw. the BSOD is not funny as you should know that it's possibly a lot of work to repair the system and the fear of eventual data loss is NOT FUNNY, too.

**Comment by bigbrotherisstillwatching** [comment]
You're right, i promise the next time i'll open a new issue. At first the BSOD scared me, because I thought about the consequences that this may have. It's ok, i read that it's just a prank.

I will test if the Dark Mode problem is gone. As for the Side-Stage problem i don't really know how to reproduce this. There are so many (new) options and i forgot what the Side-Stage is and how to display it.

Please wait with closing this issue until everything is solved.

Kind regards.

**Comment by kugiigi** [comment]
Side-stage can be toggled in `Convergence > Staged Mode`
It is only available in landscape. You can drag apps to it with 3-finger drag. 3-finger tap will show/hide it.

**Comment by bigbrotherisstillwatching** [comment]
1. I've tested it and Dark Mode stays disabled when rebooting at night, so this is fixed, i think.

2. Saw no problem with that, so it's fixed, too.

3. Does work. Fixed.

4. I don't know, because, as mentioned above, i don't use jerk-installer anymore. Instead i save /usr/share/lomiri folder and just copy everything inside the MOD folder into it. To revert i remove everything in /usr/share/lomiri, copy the backuped files into it and restart Lomiri with the UT Tweak Tool.

5. Fixed.

**Comment by kugiigi** [comment]
Thanks for the feedback. Can we close this now?
